### PR TITLE
Feature / Refactor admin pages

### PR DESCRIPTION
**Pull request by mmrshk**
Refactoring for PR that did Stas.

### CODE REVIEW

File: app/controllers/admins/users_controller.rb [hunk 1]
```diff
@@ -17,22 +17,21 @@ def new
...
-    def edit
-    end
+    def edit; end
 
     def update
-      redirect_to admins_user_url(@user), notice: '' if last_super_admin_tries_to_update_its_role?
-
-      if @user.update(user_params)
-        redirect_to admins_user_url(@user), notice: '' 
+      if last_super_admin_tries_to_update_its_role?
```
Comment (Bilanuk) [inline app/controllers/admins/users_controller.rb:31]: why did u removed guard clause? U can use multi line guard clause if there no enouth space for one-liner.

File: app/controllers/admins/users_controller.rb [hunk 2]
```diff
@@ -55,10 +54,7 @@ def last_super_admin_tries_to_update_its_role?
     end
 
     def last_super_admin?
-      @last_super_admin ||= begin
-        super_admins = User.where(role: User::USER_ROLES[:super_admin]).size 
-        super_admins < 2
-      end
+      User.where(role: User::USER_ROLES[:super_admin]).size < 2
```
Comment (Bilanuk) [inline app/controllers/admins/users_controller.rb:57]: @last_super_admin ||= User.where(role: User::USER_ROLES[:super_admin]).size < 2
?
CC @mmrshk

File: app/controllers/admins/users_controller.rb [hunk 3]
```diff
@@ -2,42 +2,45 @@
...
 module Admins
   class UsersController < AdminsController
-    before_action :user, :super_admin, only: %i[index show edit update destroy]
-    helper_method :super_admin?, :admin?, :last_super_admin?
+    before_action :user, only: %i[edit show update destroy]
+    helper_method :last_super_admin?
 
     def index
-      @users = User.all
+      @users = User.all if current_user.role_super_admin?
```
[the remark this answers is not in the document]
Comment (Bilanuk) [reply app/controllers/admins/users_controller.rb:9]: he added prefix to model. Why do u think its not his work?
But still this condition is useless. He need to implement access logic in callbacks and not inside of action.

File: app/controllers/admins/users_controller.rb [hunk 4]
```diff
@@ -2,42 +2,45 @@
...
 
     def new
       @user = User.new
-      @path = '/admins/users'
     end
 
     def create
       @user = User.new(user_params)
-      @user.role = ROW_USER
+      @user.role_user! unless current_user.role_super_admin?
```
[the remark this answers is not in the document]
Comment (stasfly) [reply app/controllers/admins/users_controller.rb:20]: line 20 protects the DB against code injection through curl POST request. Or do you count that it is impossible to immitate POST request to create action?

File: app/controllers/admins/users_controller.rb (hunk 2 above)
Comment (stasfly) [reply app/controllers/admins/users_controller.rb:57]: Agree. @last_super_admin ||= User.where(role: User::USER_ROLES[:super_admin]).size < 2 is better

File: app/controllers/admins/users_controller.rb (hunk 1 above)
Comment (stasfly) [reply app/controllers/admins/users_controller.rb:31]: Well i'm not sure that long string with flash message and condition in the end pass the rubocop.

**Review by mmrshk — COMMENTED** [review]
Approved, fix `config/en.yml` please

**Pull request merged**